scripts/config.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Project Settings
+PROJECT_NAME="FileOrganiser"
+PROJECT_DIR="$(cd "$(dirname "$0")/.." && pwd)"
+SCHEME="FileOrganiser"
```

Shellcheck (0.11.0)

[warning] SCHEME appears unused. Verify use (or export if used externally).

(SC2034)